chispa/dataframe_comparer.py

```diff
@@ -12,7 +12,7 @@ class DataFramesNotEqualError(Exception):
 
 
 def assert_df_equality(df1, df2, ignore_nullable=False, transforms=None, allow_nan_equality=False,
-                       ignore_column_order=False, ignore_row_order=False, ignore_schema=False):
+                       ignore_column_order=False, ignore_row_order=False, ignore_schema=False, precision=0):
```

Comment:
Isn't the `precision` argument only relevant for `assert_approx_df_equality` and not for `assert_df_equality`?  I'm not sure we should add this argument here.

Reply:
You are right. `precision` was not originally part of `assert_df_equality` signature. I thought of adding, just to reuse the same code in approx_df_equality. 
Made following changes now.
1. Removed precision argument from `assert_df_equality`
2. Added precision as part of if condition in the `assert_approx_df_equality`

Please check now @MrPowers